src/models/ui.ts

```diff
@@ -2,12 +2,20 @@ import { types } from "mobx-state-tree";
 
 export const UIModel = types
   .model("UI", {
-    sampleText: "Hello World"
+    sampleText: "Hello World",
+    showInvestigationModalSelect: false,
+    showInvestigationPanel: false
   })
   .actions((self) => {
     return {
       setSampleText(text: string) {
         self.sampleText = text;
+      },
+      setShowInvestigationModalSelect(val: boolean) {
```

Comment:
They'll be trivial, but would you mind adding some tests to `ui.test.ts` for these new functions?

If you don't mind, it would be helpful to get rid of uses of `sampleText` too, now that we have actual functional examples

Reply:
Yup.  No problem.